config/environments/test.rb

```diff
@@ -52,4 +52,7 @@
   config.action_mailer.delivery_method = :test
   config.action_mailer.default_options = { from: "mail@example.com" }
   config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
+  config.hosts << "check_records.lvh.me"
+  config.hosts << "qualifications.lvh.me"
```

Comment:
Do we have access to the env vars set in .env.test at this point?

Reply:
It's the protocol in the env var value that's causing an issue (see https://github.com/DFE-Digital/access-your-teaching-qualifications/pull/123#discussion_r1179014302)